### DISCUSSION TITLE
Argument id mutation in __init__

### DISCUSSION THREAD

**Discussion by dfreeman06** [pydantic/pydantic#2648] [Question] [answered]
Consider the example below. Using `Parent.construct` results in a `Parent` instance with the same `Child` instance but using the normal `Parent` `__init__` (which triggers the validations) results in a `Parent` instance with a new `Child` instance. 

Is there is a configuration option for the behavior I am looking for that will run the validation and not change the argument?

```
from pydantic import BaseModel

class Child(BaseModel):
    pass

class Parent(BaseModel):
    child: Child

child = Child()
p1 = Parent(child=child)
p2 = Parent.construct(child=child)

assert p2.child == child
assert p2.child is child

assert p1.child == child
assert p1.child is child, 'would expect same instance of child'
```

**Comment by PrettyWood** [comment, marked as answer]
Hi @dfreeman06
Since v1.8, you can set `copy_on_model_validation = False` in your config :)
See https://github.com/samuelcolvin/pydantic/discussions/2457#discussioncomment-425033
Hope this helps!

**Reply by dfreeman06** [reply]
Yes, very helpful!
